__tests__/__unit__/extension.unit.test.ts

```diff
@@ -1375,907 +1494,769 @@ describe("Extension Unit Tests", () => {
+            user: "",
+            password: "",
+            hostname: "fake",
+            port: 443,
+            protocol: "https",
+            type: "basic",
+        });
+        const dsNode = new ZoweDatasetNode("testSess", vscode.TreeItemCollapsibleState.Expanded, sessNode, sessionwocred);
+        dsNode.contextValue = globals.DS_SESSION_CONTEXT;
+        Object.defineProperty(Profiles, "getInstance", {
+            value: jest.fn(() => {
+                return {
+                    allProfiles: [{name: "firstName", profile: {user:undefined, password: undefined}}, {name: "secondName"}],
+                    defaultProfile: {name: "firstName"},
+                    promptCredentials: jest.fn(()=> {
+                        return ["fake", "fake", "fake"];
+                    }),
+                    getProfiles: jest.fn(),
+                    validProfile: ValidProfileEnum.VALID,
+                    checkCurrentProfile: jest.fn(),
+                    loadNamedProfile: mockLoadNamedProfile
+                };
+            })
+        });
 
-        await dsActions.enterPattern(favoriteSample, testTree);
+        showInputBox.mockReturnValueOnce("fake");
+        showInputBox.mockReturnValueOnce("fake");
 
-        expect(mockAddZoweSession.mock.calls.length).toBe(1);
-        expect(mockAddZoweSession.mock.calls[0][0]).toEqual("sestest");
+        await dsActions.openPS(dsNode, true, testTree);
+        expect(openTextDocument.mock.calls.length).toBe(1);
+        expect(showTextDocument.mock.calls.length).toBe(1);
     });
 
-    it("Testing that saveFile is executed successfully", async () => {
-        const testDoc: vscode.TextDocument = {
-            fileName: path.join(globals.DS_DIR, "/sestest/HLQ.TEST.AFILE"),
-            uri: null,
-            isUntitled: null,
-            languageId: null,
-            version: null,
-            isDirty: null,
-            isClosed: null,
-            save: null,
-            eol: null,
-            lineCount: null,
-            lineAt: null,
-            offsetAt: null,
-            positionAt: null,
-            getText: null,
-            getWordRangeAtPosition: null,
-            validateRange: null,
-            validatePosition: null
-        };
-        const testDoc0: vscode.TextDocument = {
-            fileName: path.join(globals.DS_DIR, "HLQ.TEST.AFILE"),
-            uri: null,
-            isUntitled: null,
-            languageId: null,
-            version: null,
-            isDirty: null,
-            isClosed: null,
-            save: null,
-            eol: null,
-            lineCount: null,
-            lineAt: null,
-            offsetAt: null,
-            positionAt: null,
-            getText: null,
-            getWordRangeAtPosition: null,
-            validateRange: null,
-            validatePosition: null
-        };
+    it("Testing that that openPS credentials prompt works with favorites", async () => {
```

Comment:
All credential checks have been covered in profiles.unit.test.ts so they can be removed